synapse/server_notices/resource_limits_server_notices.py

```diff
@@ -197,7 +197,7 @@ def _is_room_currently_blocked(self, room_id):
             room_id(str): The room id of the server notices room
 
         Returns:
-            Deferred[bool|List]:
+            Deferred[Tuple[bool|List]]:
```

Comment:
I think you want a comma between these: `Deferred[Tuple[bool, List]]` since it is a tuple of two elements, the first of which is a bool and the second is a list.